documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -362,6 +365,37 @@ ifdef::community[]
 == Confluent Schema Registry
 
 There is an alternative https://github.com/confluentinc/schema-registry[schema registry] implementation provided by Confluent.
+
+[id="overview-of-deploying-confluent-schema-registry"]
+=== Confluent Schema Registry deployment overview
+
+To install standalone Confluent Schema Registry, please see Confluent's https://docs.confluent.io/platform/current/installation/installing_cp/overview.html[official deployment guide].
+
+[NOTE]
+====
+There's also possibility to https://docs.confluent.io/platform/current/installation/docker/installation.html[install] it as a container.
+====
+
+[id="deploying-confluent-schema-registry-with-debezium-containers"]
+=== Deploying Confluent Schema Registry with {prodname} containers
+
+As of {prodname} 2.0.0, Confluent Schema Registry support is on part of the {prodname} containers.
```

Comment:
```suggestion
Beginning with {prodname} 2.0.0, Confluent Schema Registry support is included in the {prodname} containers.
```
It's not clear what you mean when you state that in 2.0 CSR support "is on part of the Debezium containers." I'm interpreting this to mean that the CSR registry is built in to the Debezium 2.0 containers, yes?

Reply:
ops, what I wrote don't make any sense. Actually I meant that the support of Confluent Schema Registry was dropped. i adjusted your suggestion.